project.yaml

```diff
@@ -420,7 +420,10 @@ Layer:
                 CASE WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text
                   ELSE 'INT-normal'::text
                 END AS service,
-                CASE WHEN oneway IN ('yes', '-1') THEN oneway ELSE NULL END AS oneway,
+                CASE WHEN oneway IN ('yes', '-1') THEN oneway
+                  WHEN (highway IN ('motorway', 'motorway_link') OR junction IN ('roundabout')) AND (oneway <> 'no' OR oneway IS NULL) THEN 'yes'
+                  ELSE NULL
+                  END AS oneway,
```

Comment:
have we been tabbing the END for a case the same as the first line of it or the same as the contents of it?